### ISSUE TITLE
A new compose isn't immediately saved to the store

### ISSUE THREAD

**Issue by ondrejbudai** [osbuild/osbuild-composer#242]
When I create a compose using cockpit-composer it's not immediately shown in the compose list. This is not a critical bug but it's slightly annoying.

I think the cause of this is that we depsolve the compose before putting it into the store. Imo we should put it in the store as soon as possible as depsolving can take a while (especially when downloading all the dnf metadata) and a user might get confused why the image isn't showing up in the list.

**Comment by teg** [comment]
Once we move over to the rpm stage, we will no longer have to clear our caches before the final depsolve, so we should revisit this issue then as that may improve the situation a lot.

**Comment by teg** [comment]
I agree, we should implement this change. My previous comment no longer applies (we moved to rpm pipelines).

**Comment by bcl** [comment]
Pretty sure this no longer applies.
